data/themes/default/style.css

```diff
@@ -620,15 +620,17 @@ TrackContentObjectView   {
 	qproperty-selectedColor: #006B65;
 	qproperty-BBPatternBackground: #373d48;
 	qproperty-textColor: #fff;
+	qproperty-textBackgroundColor: rgba(0, 0, 0, 75);
 	qproperty-textShadowColor: rgb(0,0,0,200);
```

Comment:
> Now that we actually have a shaded background, is there even a need for a shadow for the text?

Should be tested with custom colors.

![image](https://user-images.githubusercontent.com/6345473/28318756-e87644b4-6b99-11e7-91dc-8b78ce052f6f.png)